src/App.jsx

```diff
@@ -14,57 +16,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const products = goods.map((item, index) => ({
+  productsId: index,
+  productName: item,
+}));
```

Comment:
You're using the index as a key in `productsId` (checklist item 2). This is not recommended because the key should be stable, predictable, and unique for each element in the list. If the order of items changes, it can lead to bugs and performance issues. A better option would be to use unique identifiers for your products.